packages/components/src/molecules/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,162 @@
+import React, {
+  type ReactNode,
+  useState,
+  forwardRef,
+  type HTMLAttributes,
+  useRef,
+  useEffect,
+} from 'react'
+import Icon from '../../atoms/Icon'
+import IconButton from '../IconButton'
+/**
+ * Specifies tooltip position.
+ */
+export type Side = 'top' | 'right' | 'bottom' | 'left'
+
+/**
+ * Specifies tooltip alignment.
+ */
+export type Alignment = 'start' | 'center' | 'end'
+
+/**
+ * Combines side + alignment (e.g., "top-start").
+ */
+export type Placement = `${Side}-${Alignment}`
+
+export interface TooltipProps
+  extends Omit<HTMLAttributes<HTMLDivElement>, 'content'> {
+  /**
+   * Text/content of the tooltip.
+   */
+  content: ReactNode
+  /**
+   * Defines the side or side-alignment (e.g., "top-center", "right-end") of the tooltip.
+   */
+  placement?: Placement
+  /**
+   * If the tooltip can be closed by a button.
+   */
+  dismissible?: boolean
+  /**
+   * Called when the dismiss button is clicked.
+   */
+  onDismiss?: (
+    ev:
+      | React.KeyboardEvent<HTMLDivElement>
+      | React.MouseEvent<HTMLButtonElement>
+  ) => void
+  /**
+   * Element that activates the tooltip on hover/focus.
+   */
+  children: ReactNode
+  /**
+   * ID to find this component in testing tools (e.g.: cypress, testing library, and jest).
+   */
+  testId?: string
+  /**
+   * Maximum width of the tooltip.
+   */
+  maxWidth?: number
+  /**
+   * ID for the tooltip content to be used with aria-describedby.
+   */
+  describedById?: string
+}
+
+const Tooltip = forwardRef<HTMLDivElement, TooltipProps>(function Tooltip(
+  {
+    content,
+    placement = 'top-center',
+    dismissible = false,
+    onDismiss,
+    children,
+    testId = 'fs-tooltip',
+    maxWidth = 300,
+    describedById = 'tooltip-content',
+    ...otherProps
+  },
+  ref
+) {
+  const [open, setOpen] = useState(false)
+  const [dismissed, setDismissed] = useState(false)
+  const dismissButtonRef = useRef<HTMLButtonElement>(null)
+  const triggerRef = useRef<HTMLDivElement>(null)
+
+  const handleDismiss = (
+    ev:
+      | React.KeyboardEvent<HTMLDivElement>
+      | React.MouseEvent<HTMLButtonElement>
+  ) => {
+    onDismiss?.(ev)
+    setOpen(false)
+    setDismissed(true)
+  }
+
+  const toggleOpen = () => {
+    if (dismissed) {
+      setDismissed(false)
+    }
+    setOpen(true)
+  }
+
+  const handleKeyDown = (event: React.KeyboardEvent<HTMLDivElement>) => {
+    if (event.key === 'Escape') {
+      handleDismiss(event)
+    }
+  }
+
+  useEffect(() => {
+    if (open && dismissible) {
+      dismissButtonRef.current?.focus()
+    }
+  }, [open, dismissible])
+
+  return (
+    <div
+      data-fs-tooltip-wrapper
```

Comment:
```suggestion
      data-fs-tooltip
```

I know this might be a bit confusing, but I think it's best to maintain the existing pattern. The outer element should be named data-fs-component-name. Can you please, rename this element to `data-fs-tooltip`, and rename the current `data-fs-tooltip` to `data-fs-tooltip-wrapper`? Also update in the stylesheet.

Reply:
Done ✅